### ISSUE TITLE
508-defect-3 [COGNITION]: System Landing - Building image should have ALT text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21077]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Angela or Trevor_

## User Story or Problem Statement

As a screen reader user, I would like to hear an aural (spoken) description of the duotone building image on the system front page.

## Details

The duotone blue image of the building adds a richness to the system landing page. It would be good to provide a spoken equivalent for screen reader users. It could be as simple as "Front of the Jack C. Montgomery Department of Veterans Affairs Medical Center building from the east parking lot."

## Acceptance Criteria

- [ ] Alt text is added to the banner image
- [ ] Zero axe-core violations appear after the alt text is added

## Environment
* https://staging.va.gov/eastern-oklahoma-health-care/

## Proposed Solution (if known)

```diff
<img
  src="https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/7_2_medium_thumbnail/public/2021-01/4.jpg"
- aria-hidden="true"
- role="presentation"
- alt=""
+ alt="Front of the Jack C. Montgomery Department of Veterans Affairs Medical Center building from the east parking lot."
width="100%">
```

## WCAG or Vendor Guidance (optional)
* [Non-text Content: Understanding SC 1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)

## Screenshots or Trace Logs
<img width="453" alt="Screen Shot 2021-03-05 at 4 46 53 PM" src="https://user-images.githubusercontent.com/934879/110182747-04fa0e00-7dd3-11eb-9990-d62a22443c97.png">
